search/package.json

```diff
@@ -4,15 +4,23 @@
   "private": true,
   "dependencies": {
     "@appbaseio/reactivesearch": "^3.12.2",
-    "@material-ui/core": "^4.11.0",
-    "@material-ui/icons": "^4.9.1",
-    "@material-ui/styles": "^4.11.1",
+    "@jsonforms/core": "^2.5.0",
+    "@jsonforms/material-renderers": "^2.5.0",
+    "@jsonforms/react": "^2.5.0",
+    "@material-ui/core": "^4.11.2",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/styles": "^4.11.3",
     "@nuxtjs/axios": "^5.12.2",
     "@testing-library/dom": "^5",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "@types/jest": "^26.0.20",
+    "@types/node": "^14.14.22",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
     "js-cookie": "^2.2.1",
+    "js-yaml": "^4.0.0",
     "node-fetch": "^2.6.1",
     "node-forge": "^0.10.0",
     "react": "^16.13.1",
```

Comment:
Somehow the react-jsonschema-form dependencies are still here, when they were removed in the initial PR. Was this due to an incorrect resolution of a merge conflict?